### ISSUE TITLE
(AM)OLED Power Saving mode

### ISSUE THREAD

**Issue by ostrosablin** [gurrhack/ForkFront-Android#1]
Currently, NetHack for Android displays area outside map in gray. But OLED screens have an advantage - they don't waste power to display true black pixels (#000000). It would be great if Android NetHack port could make use of this advantage to squeeze extra battery life from such devices. It would work great with ASCII mode, because in this mode, it could keep around 70-80% of screen completely blacked and could lead to a huge battery life boost. I think in this mode, a border could be drawn around the map area to separate it from black background. If it's possible to invert colors of buttons on panel (black buttons with white labels and outlines) it could help even further.

**Comment by gurrhack** [comment]
Good idea. I will most likely change the "outside" to a border for the next release. And also add an option to disable it, making it slightly more challenging to explore the dungeon.
I'm also changing the button style, but maybe not for the next release. The bright buttons have a tendency to burn into the display if you play a lot. Black buttons would help with that as well.
